src/components/Comments.jsx

```diff
@@ -36,8 +65,14 @@ function Comments(props) {
   } else {
     return (
       <div className="comments">
-        {comments.map((comment) => (
-          <Comment key={comment.comment_id} data={comment} />
+        {successMessage && (
+          <strong>
+            <p className="success-message">{successMessage}</p>
+          </strong>
+        )}
+        {<AddCommentForm action={handleNewComment} id={id} />}
+        {comments.map((comment, i) => (
+          <Comment key={`${comment} - ${i}`} data={comment} />
```

Comment:
Each comment should have a unique ID from the database anyway and would be ideal for the key